apps/entity-service/src/file/file-upload.service.ts

```diff
@@ -98,42 +109,41 @@ export class FileUploadService {
       createdBy: this.entitiesService.userId
     };
 
-    const dbMedia = await Media.create(media as Media);
+    console.log(media);
 
-    return new MediaDto(dbMedia, {
-      url: this.mediaService.getUrl(dbMedia),
-      thumbUrl: this.mediaService.getUrl(dbMedia, "thumbnail"),
-      entityType: entity,
-      entityUuid: model.uuid
-    });
+    const dbMedia = new Media(media as Media);
+    return dbMedia.save();
   }
 
-  private getConfiguration(entity: EntityClass<EntityModel>, collection: string): MediaConfiguration {
-    const configuration = (entity as any).MEDIA[collection];
+  private getConfiguration(entityModel: EntityClass<EntityModel>, collection: string): MediaConfiguration {
+    // TODO: ask Nathan what is the best way to check if a EntityClass has a MEDIA property and how to type it
```

Comment:
For this, I would accept a specific type of model in the method declaration. See `types/util.ts` in the DB lib for an example of this pattern, and find their usages. I'm thinking something specifically similar to `LaravelModelCtor`.